MiniSpace.APIGateway.Ocelot/src/MiniSpace.APIGateway.Ocelot/Infrastructure/CorrelationContext.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+
+namespace MiniSpace.APIGateway.Ocelot.Infrastructure
+{
+    internal class CorrelationContext
+    {
+        public string CorrelationId { get; set; }
+        public string SpanContext { get; set; }
+        public UserContext User { get; set; }
+        public string ResourceId { get; set; }
+        public string TraceId { get; set; }
+        public string ConnectionId { get; set; }
+        public string Name { get; set; }
+        public DateTime CreatedAt { get; set; }
+
+        public class UserContext
+        {
+            public string Id { get; set; }
+            public bool IsAuthenticated { get; set; }
+            public string Role { get; set; }
+            public IDictionary<string, string> Claims { get; set; }
+        }
+    }
```

Comment:
You are not currently using the Ocelot.

Reply:
Yeap, we are currently not planning the new framework for Convey and ApiGateway pattern still.